GCGS-Central-Digital-Platform.slnLaunch

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "Name": "New Profile",
+    "Projects": [
+      {
+        "Name": "Services\\CO.CDP.Tenant.WebApi\\CO.CDP.Tenant.WebApi.csproj",
+        "Action": "Start",
+        "DebugTarget": "http-for-docker"
+      },
+      {
+        "Name": "Frontend\\CO.CDP.OrganisationApp\\CO.CDP.OrganisationApp.csproj",
+        "Action": "Start",
+        "DebugTarget": "http-for-docker"
+      },
+      {
+        "Name": "Libraries\\CO.CDP.Authentication\\CO.CDP.Authentication.csproj",
+        "Action": "Start"
+      }
+    ]
+  }
+]
```

Comment:
Since this profile only starts a subset of services, seems like it's made for one specific use case. Does it deserve to be comitted to the repo?